internal/staticupload/staticupload.go

```diff
@@ -171,6 +181,10 @@ func (c *Client) invalidateCacheForKeys(ctx context.Context, keys []string) (str
 		return "", InvalidationError{inner: fmt.Errorf("too many keys to invalidate: %d", len(keys))}
 	}
 
+	if len(keys) == 0 {
+		return "", InvalidationError{inner: errors.New("no keys to invalidate")}
+	}
+
```

Comment:
I would assume the cdn client errors anyway if we pass an empty list of keys?
In that case we wouldnt need to manually return an error here. 
Of course this depends on if the error has side effects in the cdn or similar